packages/react-ui-components/src/utils/useFont/useFontsApi.ts

```diff
@@ -0,0 +1,21 @@
+import { useCallback, useMemo } from 'react';
+
+const FONT_NOT_FOUND_ERROR = `Font was not found, make sure the font is included in your css`;
+
+export const useFontsApi = (fontFamily: string, fontSize: number) => {
+	const fontName = useMemo(() => `${fontSize}px ${fontFamily}`, [fontFamily, fontSize]);
+
+	const checkFont = useCallback(() => document.fonts.check(fontName), [fontName]);
+
+	const loadFont = useCallback(async () => {
+		if (checkFont()) return;
```

Comment:
Shouldn't it be? `if (!checkFont()) return;`?

Reply:
`checkFont` verifies if the font has already been loaded, if it returns `true`, there's no need for this function's execution to continue